common/test/acceptance/pages/lms/video/video.py

```diff
@@ -566,21 +568,36 @@ def select_transcript_format(self, transcript_format):
             bool: Selection Result.
 
         """
-        button_selector = self.get_element_selector(VIDEO_MENUS['transcript-format'])
+        button_selector = self.get_element_selector(VIDEO_MENUS['transcript-format']) # .video-tracks .button-more.has-dropdown
 
         button = self.q(css=button_selector).results[0]
 
         coord_y = button.location_once_scrolled_into_view['y']
         self.browser.execute_script("window.scrollTo(0, {});".format(coord_y))
 
-        hover = ActionChains(self.browser).move_to_element(button)
-        hover.perform()
+        # New markup/button no longer functions on hover, but on click
+        # Commenting out for now; need to update or remove.
 
-        if '...' not in self.q(css=button_selector).text[0]:
-            return False
+        # hover = ActionChains(self.browser).move_to_element(button)
+        # hover.perform()
+
+        # Since we click now, updating to click
```

Comment:
I was providing my thought process until Andy or yourself could review this to make sure I wasn't crazy :) Cleaning it up.